benchmark/cglm/README.md

```diff
@@ -0,0 +1,89 @@
+# CGLM Dataset
+
+In this directory, you can find the files necessary to run experiments with the CGLM dataset. 
+This dataset contains pictures of landmarks (e.g., the Montblanc) over time.
+The dataset was introduced [in this paper](https://drimpossible.github.io/documents/ACM.pdf).
+However, their host is down and we have to regenerate the dataset.
+It makes use of the [Google Landmarks V2 dataset](https://github.com/cvdfoundation/google-landmark).
+
+## Generating the dataset for Modyn
+
+As a first step, you will need to download the Google Landmarks v2 Images.
+You can check out [their repository](https://github.com/cvdfoundation/google-landmark) for instructions, or try this snippet:
+
+```bash
+wget -c https://raw.githubusercontent.com/cvdfoundation/google-landmark/master/download-dataset.sh
+mkdir cglm && cd cglm
+bash ../download-dataset.sh train 499
+```
+
+Next, we need the image metadata.
+We have pregenerated and host the metadata on Github for convenience.
+If you want to check out how we generate this metadata, check out the next file.
+
+```bash
+wget https://github.com/eth-easl/cglm-metadata/raw/main/cglm_labels_timestamps_clean.csv
```

Comment:
It might make sense to store the fully scraped dataset (before dataframe transformations) in public nfs share of the sgs cluster.

Reply:
Will do!